docs/develop/add-in-manifests.md

```diff
@@ -2,23 +2,23 @@
 title: Office Add-ins manifest
 description: Get an overview of the Office Add-in manifest and its uses.
 ms.topic: overview
-ms.date: 07/27/2023
+ms.date: 03/15/2024
 ms.localizationpriority: high
 ---
 
 # Office Add-ins manifest
 
 Every Office add-in has a manifest. There are two types of manifests:
 
-- **XML manifest:** This is the only type of manifest that is currently supported for production add-ins. As the name indicates, it is XML format. This type of manifest can't be used for an app that combines an add-in with some other kind of Teams App; that is, some other kind of extension of the Microsoft 365 platform.
-- **unified manifest for Microsoft 365:** This is a JSON-formatted manifest that has been used for years as the manifest for Teams Apps. It is supported for add-ins only as a preview currently, and only on Outlook for Windows. It shouldn't be used with a production add-in. When it releases to general availability, add-ins that use this manifest can be combined with other kinds of Teams Apps in a single app that is installable as a unit whole. 
+- **XML manifest:** This is the only type of manifest that is currently supported for non-Outlook add-ins. As the name indicates, it is XML format. This type of manifest can't be used for an app that combines an add-in with some other kind of extension of the Microsoft 365 platform.
+- **unified manifest for Microsoft 365:** This is an expanded version of the JSON-formatted manifest that has been used for years as the manifest for Teams Apps. Currently, this manifest is only available for Outlook add-ins and only in Office linked to a Microsoft 365 subscription and installed on Windows, on a mobile device, or in Outlook on the web. We're working on extending support to Excel, PowerPoint, and Word, as well as to Outlook on Mac, and to perpetual versions of Office. Add-ins that use this manifest can be combined with other kinds of extensions of the Microsoft 365 platform in a single app that is installable as a unit whole. 
```

Comment:
```suggestion
- **XML manifest**: This is the only type of manifest that is currently supported for non-Outlook add-ins. As the name indicates, it's XML format. This type of manifest can't be used for an app that combines an add-in with some other kind of Microsoft 365 platform extension.
- **unified manifest for Microsoft 365**: This is an expanded version of the JSON-formatted manifest that has been used for years as the manifest for Teams Apps. Currently, this manifest is only available for Outlook add-ins and only in Office linked to a Microsoft 365 subscription and installed on Windows, on a mobile device, or in Outlook on the web. We're working on extending support to Excel, PowerPoint, and Word, as well as to Outlook on Mac and to perpetual versions of Office. Add-ins that use this manifest can be combined with other kinds of Microsoft 365 platform extensions in a single app that's installable as a unit whole. 
```

Reply:
> 

I may be an old fogy about stacked nouns, but I'm making all of these changes except that I'm going to leave the prepositional phrase "extensions of the Microsoft 365 platform" rather than stack a 4th noun with "Microsoft 365 platform extensions".